apps/component-generator/component-templates/ComponentTemplate/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluentui-react-native/component-name",
+  "version": "0.15.8",
```

Comment:
Should probably start with version 0.0.1 or something that's similarly "new"

Reply:
fixed